src/app/pages/organization/companies/company/company.component.ts

```diff
@@ -50,15 +50,14 @@ export class CompanyComponent implements OnInit {
     private dialogService: DialogService,
     private translateService: TranslateService,
     private router: Router) {
+    this.canUpdateCompany = false;
```

Comment:
You can set it to false at declaration level:

`public canUpdateCompany = false;`